examples/eventwebhook/consumer/Src/EventWebhook/Views/EventWebhook/Index.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿
+@{
+    ViewData["Title"] = "Index";
+}
+
+<h1>You have successfully launched the server!</h1>
+<p>
+    Check out <a href="https://github.com/sendgrid/sendgrid-csharp/tree/master/examples/eventwebhook/consumer">the documentation</a> on how to use this webhook to utilize the SendGrid Event webhook.
```

Comment:
Switch `master` to `HEAD` as we'll be changing the default branch name soon.

Reply:
done